cypress/integration/context-selection/categories/index.js

```diff
@@ -159,3 +184,14 @@ Then('the selector should show that some items have been selected', () => {
 Then('a message is being displayed in the data workspace', () => {
     cy.contains('The current selection does not have a form').should('exist')
 })
+
+Then('a message is being displayed in the data workspace', () => {
+    cy.contains('The current selection does not have a form').should('exist')
+})
```

Comment:
Duplicate of the same step definition right above this one

Reply:
fixed. sorry